### ISSUE TITLE
Add option to start daemon without   schedules

### ISSUE THREAD

**Issue by soloam** [Flexget/Flexget#2682]
### Expected behaviour:

A option to start daemon without the schedules, this would allow using the api, or do some test, without having to comment out the schedules if they are set

### Actual behaviour:

When we what to use the api, we have to start the daemon, with this the schedules are set, it would be nice to have a modifier to the damon to not start them. Like flexget daemon start -d --no-schedules

Would this be a big change? Is it possible?

Thank you

**Comment by liiight** [comment]
Hmmm, this is a very interesting idea. So the use case is that you want to have schedules but you also want the ability to start the WEBUI/API without triggering them, correct?

**Comment by soloam** [comment]
Basically yes. In my case, I have the schedules set, but If I what to test the api and not schedules, I have to comment out the schedules!

I noticed this because I tried to do a symbolic link from my config to start a new instance of the flexget with only the api, to do some experiments, but I didn't what the schedules to run. But I also wanted to use the same config file because I wanted to make changes and replicate them to the main instance. I also wanted to avoid copy paste, to minimize forgetting something and losing hours finding what was it!

So, like we do flexget daemon start -d, to start the daemon in background, we could have something like flexget daemon start -d --no-schedule, or flexget daemon start -ns,

**Comment by liiight** [comment]
First off, from a technical stand point, it is possible and doesnt look that painful to do.
That being said, I'm not sure it's worth it. 
Flexget is an automator, one that supposed to constantly run and do... stuff. What is the downside of tasks running even when you just want to check out the API? What is the reason you want to disable the schedules?

**Comment by soloam** [comment]
The main reason is that I want to automate the download on request, like, a user asks for a movie and it searches, downloads and moves and renames it! If a task is running because of scheduling, it will make that I will have to wait until it ends! My first ideia is to have a symbolic link from the config to a new folder and have it start a new database just for this. And way a link? Because I don't want to duplicate all corrections and improvements in both configs! All works ok, the only thing missing is that in the config I have the schedules, and that will make it run in both instances!

**Comment by paranoidi** [comment]
Yeah, that waiting is annoying. I think the best answer would be to start executing that manual task immediately in parallel thread. SQLite is just PITA to deal with that ...maybe we could try it experimentally and see how it goes? Assuming someone is interested adding the support for it :)

**Comment by gazpachoking** [comment]
> I think the best answer would be to start executing that manual task immediately in parallel thread. SQLite is just PITA to deal with that ...maybe we could try it experimentally and see how it goes?

😬
